doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+paths to the diffusion input file, b-values file, b-vectors file and mask file,
+followed by optional arguments. In this case, we will be specifying the list of
+metrics to save (``save_metrics``), the output directory (``out_dir``), and the
+name of the tensors volume to be saved (``out_tensor``).
+
+We will run the command as::
+
+    dipy_fit_dti data/stanford_hardi/HARDI150.nii.gz data/stanford_hardi/HARDI150.bval data/stanford_hardi/HARDI150.bvec stanford_hardi_mask/brain_mask.nii.gz --save_metrics "md" "mode" "tensor" --out_dir "recons_dti_output" --out_tensor "dti_tensors.nii.gz"
+
+This command will save the DTI metrics to the specified output directory.
+
+--------------------------
+Diffusion Kurtosis Imaging
+--------------------------
+
+The diffusion kurtosis model is an expansion of the diffusion tensor model. In
+addition to the diffusion tensor (DT), the diffusion kurtosis model quantifies
+the degree to which water diffusion in biological tissues is non-Gaussian using
+the kurtosis tensor (KT) [Jensen2005]_.
+
+Measurements of non-Gaussian diffusion from the diffusion kurtosis model are of
+interest because they can be used to charaterize tissue microstructural
+heterogeneity [Jensen2010]_. 
+
+Moreover, DKI can be used to:
+
+* Derive concrete biophysical parameters, such as the density of axonal fibers and diffusion tortuosity [Fierem2011]_
+
+* Resolve crossing fibers in tractography and to obtain invariant rotational measures not limited to well-aligned fiber populations [NetoHe2015]_.
+
+We will use the ``cfin_multib`` dataset in DIPY to showcase this reconstruction
+method. You can also use your own data!
+
+Create the directories in which to save the DKI metrics (e.g.: ``recons_dki_output``)
+and mask file (e.g.: ``cfin_multib_mask``)::
+
+    mkdir recons_dki_output cfin_multib_mask
+
+The Diffusion Kurtosis Imaging reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+parameters can also be provided. In this case, we will be specifying the threshold
+used to find b=0 directions (``b0_threshold``) and the output directory (``out_dir``). 
+
+To get the mask file, run the command as::
+
+    dipy_median_otsu data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii --vol_idx 3 4 5 6 --out_dir "cfin_multib_mask"
+
+To run the DKI reconstruction method on the data, execute the ``dipy_fit_dki``
+command, e.g.::
+
+    dipy_fit_dki data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.bval data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.bvec cfin_multib_mask/brain_mask.nii.gz --b0_threshold 70.0 --out_dir recons_dki_output
+
+This command will save the DKI metrics to the specified output directory.
+
+--------------------
+Constant Solid Angle
+--------------------
+
+We are using ``stanford_hardi`` dataset. As with any other workflow in DIPY, you can
```

Comment:
We will be using the {double_backticks}stanford_hardi{double_backticks} dataset to showcase the CSA reconstruction method.

Reply:
Reworded in 4ce067c